radix-engine/src/system/system.rs

```diff
@@ -306,14 +317,14 @@ where
             self.api
                 .kernel_get_system_state()
                 .system
-                .blueprint_schema_cache
+                .blueprint_cache
```

Comment:
Minor: slightly prefer `schema/code_cache` based on understanding from our discussion today.

Reply:
Will keep this for now as I have a separate PR which updates all naming.